fpdf/fpdf.py

```diff
@@ -449,21 +449,16 @@ def get_string_width(self, s, normalized=False):
         return w * self.font_size / 1000
 
     def set_line_width(self, width):
-        "Set line width"
+        """Set line width"""
         self.line_width = width
         if self.page > 0:
             self._out(f"{width * self.k:.2f} w")
 
     @check_page
     def line(self, x1, y1, x2, y2):
-        "Draw a line"
+        """Draw a line"""
         self._out(
-            "{:.2f} {:.2f} m {:.2f} {:.2f} l S".format(
-                x1 * self.k,
-                (self.h - y1) * self.k,
-                x2 * self.k,
-                (self.h - y2) * self.k,
-            )
+            f"{x1 * self.k:.2f} {(self.h - y1) * self.k:.2f} m {x2 * self.k:.2f} {(self.h - y2) * self.k:.2f} l S"
```

Comment:
I deliberately converted this (and a few other occurrences) from `%` to `.format` and left the arguments in their form because the resulting f-string is over 110 chars long. `black` in the CI would leave this because it is one string it cannot split automatically, but still much longer than its standard 88 chars.

Reply:
@eumiro Thanks for pointing out. Seems like I've missed two of the long f-strings. I've now broken them apart so that they as well fit into the standard 88 chars.